www/templates/index.html

```diff
@@ -127,7 +127,7 @@
                         },
                         options: {
                             responsive: true,
-                            legend: {
+     >>>>>>> dev                       legend: {
```

Comment:
Looks like you introduced three merge conflict artifacts here that need to be removed.